### DISCUSSION TITLE
Frontend build directory not found at 'E:\open-webui\build'. Serving API only.

### DISCUSSION THREAD

**Discussion by zouzeTG** [open-webui/open-webui#3786] [Q&A]
Hi, thank you for your great work ! 
How can I resolve this situation : "Frontend build directory not found at 'E:\open-webui\build'. Serving API only ?"

![image](https://github.com/open-webui/open-webui/assets/165768738/5e8f5489-d40d-420c-9b60-6328637f33ec)

Last version of Open Webui  :v0.3.8

**Comment by zouzeTG** [comment]
Precision : I start Open-webui without docker , start windows.bat

**Comment by zouzeTG** [comment]
thank

**Comment by atgehrhardt** [comment]
You are only starting the backend. You need to separately start the front end by running npm run dev in the root folder.

You'll likely need to install the does with npm install first.

**Comment by Kingbadger3d** [comment]
If you make changes to the GUI your need to run npm i then run npm run build in the open-webui directory to update code changes.
